### PR TITLE
[ThemeBundle] Ensuring that translation files are loaded in correct order

### PR DESCRIPTION
**Pull request by pamil**
| Q             | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| BC breaks?    | no
| Deprecations? | no
| Fixed tickets | #3934
| License       | MIT

### CODE REVIEW

File: src/Sylius/Bundle/ThemeBundle/Translation/Finder/OrderingTranslationFilesFinder.php [hunk 1]
```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace Sylius\Bundle\ThemeBundle\Translation\Finder;
```
Comment (stloyd) [inline src/Sylius/Bundle/ThemeBundle/Translation/Finder/OrderingTranslationFilesFinder.php]: Missing license note.

File: src/Sylius/Bundle/ThemeBundle/spec/Translation/Finder/OrderingTranslationFilesFinderSpec.php [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+class OrderingTranslationFilesFinderSpec extends ObjectBehavior
+{
+    function let(TranslationFilesFinderInterface $translationFilesFinder)
+    {
+        $this->beConstructedWith($translationFilesFinder);
+    }
+
+    function it_is_initializable()
+    {
+        $this->shouldHaveType('Sylius\Bundle\ThemeBundle\Translation\Finder\OrderingTranslationFilesFinder');
```
Comment (stloyd) [inline src/Sylius/Bundle/ThemeBundle/spec/Translation/Finder/OrderingTranslationFilesFinderSpec.php]: `OrderingTranslationFilesFinder::class` same below.

Comment (pjedrzejewski) [inline src/Sylius/Bundle/ThemeBundle/spec/Translation/Finder/OrderingTranslationFilesFinderSpec.php]: We don't do that for class it is being described.

**Comment by pjedrzejewski** [comment]
Thanks Kamil!

**Pull request merged**